### PR TITLE
PMM-6014 Push RC Client packages to Experimental Repo

### PR DESCRIPTION
**Pull request by puneet0191**

### CODE REVIEW

**Review by denisok — APPROVED** [review]

File: pmm/pmm2-server-release.groovy [hunk 1]
```diff
@@ -13,7 +13,7 @@ pipeline {
     parameters {
         choice(
             // default is choices.get(0) - laboratory
-            choices: ['laboratory', 'testing'],
+            choices: ['laboratory', 'experimental'],
```
Comment (EvgeniyPatlan) [inline pmm/pmm2-server-release.groovy:16]: let's just add experimental and not remove testing

**Review by EvgeniyPatlan — APPROVED** [review]

File: pmm/pmm2-server-release.groovy (hunk 1 above)
Comment (denisok) [reply pmm/pmm2-server-release.groovy:16]: +1

Comment (puneet0191) [reply pmm/pmm2-server-release.groovy:16]: okay will do that

Comment (puneet0191) [reply pmm/pmm2-server-release.groovy]: Done

**Review by denisok — APPROVED** [review]

**Pull request merged**